com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDRaytracingManager.cs

```diff
@@ -221,8 +221,8 @@ void RaytracingManagerCleanupNonRenderGraphResources()
                     // Grab the material for the current sub-mesh
                     Material currentMaterial = materialArray[meshIdx];
 
-                    // The material is transparent if either it has the requested keyword or is in the transparent queue range
-                    if (currentMaterial != null)
+                    // Make sure that the material is both non-null and non-decal
+                    if (currentMaterial != null && !DecalSystem.IsHDRenderPipelineDecal(currentMaterial))
```

Comment:
the correct functoin to use is IsDecalMaterial, this function IsHDRenderPipelineDecal will not work with decal shader graph